vimiv/gui/metadatawidget.py

```diff
@@ -123,7 +123,7 @@ def metadata_list_keys(self, n_cols: int = 3, to_term: bool = False):
 
             keys = sorted(set(self.handler.get_keys()))
             if to_term:
-                print(*keys, sep="\n")
+                api.print_stdout(keys)
```

Comment:
As I would not expose the command as mentioned before, this would just have to be redone...